pom.xml

```diff
@@ -187,6 +187,11 @@
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>jackson2-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>pipeline-utility-steps</artifactId>
+            <scope>test</scope>
```

Comment:
Is this just for the `touch` step? For a cross-OS replacement, try

```groovy
writeFile file: 'x.xml', text: readFile('x.xml')
```

I guess so long as this plugin is in the BOM it is not a burden to use it in tests, though.

Reply:
`touch` is cross-OS